recipes/boost/all/test_package/conanfile.py

```diff
@@ -46,6 +46,7 @@ def generate(self):
         tc.variables["WITH_STACKTRACE"] = not self.dependencies["boost"].options.without_stacktrace
         tc.variables["WITH_STACKTRACE_ADDR2LINE"] = self.deps_user_info["boost"].stacktrace_addr2line_available
         tc.variables["WITH_STACKTRACE_BACKTRACE"] = self._boost_option("with_stacktrace_backtrace", False)
+        tc.variables["WITH_URL"] = not self._boost_option("without_url", True)
```

Comment:
once rebased, this needs
```suggestion
        tc.cache_variables["WITH_URL"] = not self._boost_option("without_url", True)
```

good catch noticing that this needs to evaluate the option via `_boost_option` :)

Reply:
Thank you. Done.